.github/workflows/sonarqube-scan.yml

```diff
@@ -0,0 +1,23 @@
+on:
+  pull_request:
+    types: [opened, synchronize]
+
+name: SonarQube scan
+jobs:
+  sonarqube_scan:
+    name: SonarQube Scan
+    runs-on: ubuntu-latest
+    steps:
```

Comment:
good to have extra step for this job, where if the sonar step fails, then log a static message explaining why it's blocking the merge